app/helpers.php

```diff
@@ -33,6 +34,18 @@ function edit_image($image, $coords)
 }
 
 /**
+ * Query for total reactions for a post.
+ *
+ * @param int $postableId
+ * @param int $postableType
+ * @return int total count
+ */
+function getTotalPostReactions($postableId, $postableType)
+{
+    return Reaction::where(['postable_id' => $postableId, 'postable_type' => $postableType])->count();
```

Comment:
Actually, I'm thinking now that there's an issue with the relations between the Reaction and the Post?